README.md

```diff
@@ -1,5 +1,35 @@
 # dotnetCampus.CommandLineArgsViewer
 
+You can view all the arguments passed from shells to this tool which helps you to debug command-lines.
```

Comment:
This is a tool to debug command-lines that you can view all the arguments passed from shells.